packages/List/src/list-item.scss

```diff
@@ -290,6 +290,10 @@
         }
     }
 
+    .ids-visual-identifier__visual {
```

Comment:
It seems like a odd place to put the fix?

the list item has the problem, so you use css to target the styling of the visual-identifier component.
Why not handle this at the source? either in the visual identifier component, or in the icons directly?

Reply:
I'm not sure I agree with this. For one, maybe it's not always the case, plus you never know what they'll use for icons. We don't have slots like other design systems.
Maybe it could be handled with Visual Identifier, but currently there is no css there. Plus it's only used inside lists and tags I believe. It's an internal component.

Reply:
It makes sense to never shrink a visual identifier.